src/components/UserHoverCard.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <Transition
+    v-if="hover"
+    name="fade"
+  >
+    <div
+      v-show="show && isActive"
+      class="modal-container"
+      @click.stop
+      @mouseover="mousestay"
+      @mouseleave="mouseleave"
+    >
+      <UserInfo :address="address" />
+    </div>
+  </Transition>
+</template>
+
+<script>
+import UserInfo from './UserInfo.vue';
+
+export default {
+  components: {
+    UserInfo,
+  },
+  props: {
+    address: { type: String, required: true },
+    hover: Boolean,
+    show: Boolean,
+    isActive: Boolean,
+    mouseover: { type: Function, required: true },
+    mouseleave: { type: Function, required: true },
+    mousestay: { type: Function, required: true },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.modal-container {
+  top: 60px;
+  width: 75%;
+  border: 1px solid $card_border_color;
+  border-radius: 0.5rem;
+  position: absolute;
+  z-index: 99;
+  overflow: hidden;
+  cursor: default;
+  box-shadow: 4px 4px 4px 0 rgba(0, 0, 0, 0.3);
+  background-color: $thumbnail_background_color;
+}
+
+.fade-enter-active,
+.fade-leave-active {
```

Comment:
I'm suggesting to use scss nesting more actively:
```suggestion

  &.fade-enter-active,
  &.fade-leave-active {
```

Reply:
Including other places